backend/src/test/java/de/trailmate/backend/controller/TourControllerTest.java

```diff
@@ -0,0 +1,37 @@
+package de.trailmate.backend.controller;
+
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class TourControllerTest {
+    @Autowired
+    MockMvc mockMvc;
+
+
+
+    @Test
+
+    void getAllTours() throws Exception {
```

Comment:
naming of the method could be more precise:
for example: whenGetAllTours_ThenReturnEmptyList